src/translations/id.lua

```diff
@@ -0,0 +1,175 @@
+		"\t• Ketik <B>!modes</B> untuk mempelajari mode lain game ini.",
+	kill = "<R>%s<FC> membunuh %s",
+
+	gameModes = "<font size='-2'><FC>[<J>•</J>] Mode game baru:\n" ..
+		"\t • Module lag? Coba versi ringannya di /room #powers0lagmode\n" ..
+		"\t • Ingin naik level lebih cepat? Coba mode pemula (khusus level rendah) di /room #powers0noobmode\n" ..
+		"\t • Terlalu pro? Tantang dirimu di mode pro (khusus level tinggi) di /room #powers0promode\n" ..
+		"\t • Tidak peduli status dan ingin mencoba semua kekuatan? Coba mode bebas di /room #powers0freemode</FC></font>",
+
+	-- Victory
+	mentionWinner = "<FC>%s<FC> memenangkan ronde ini!",
+	noWinner = "<FC>Tidak ada pemenang di ronde ini. :(",
+
+	-- Powers
+	powers = {
+		lightSpeed = "Kecepatan Cahaya",
+		laserBeam = "Sinar Laser",
+		wormHole = "Lubang Cacing",
+		doubleJump = "Lompat Ganda",
+		helix = "Spiral",
+		dome = "Kubah",
+		lightning = "Petir",
+		superNova = "Supernova",
+		meteorSmash = "Bantingan Meteor",
+		gravitationalAnomaly = "Anomali Gravitasi",
+		deathRay = "Sinar Kematian",
+		atomic = "Atomic",
+		dayOfJudgement = "Hari Penghakiman",
+		waterSplash = "Cipratan Air"
+	},
+	powersDescriptions = {
+		lightSpeed = "Menggerakkan tikusmu dalam kecepatan cahaya, mendorong semua musuh.",
+		laserBeam = "Menembakkan sinar laser yang sangat kuat sehingga musuh bisa merasakannya.",
+		wormHole = "Men-teleportasikan tikusmu ke depan melalui Lubang Cacing.",
+		doubleJump = "Melakukan lompat ganda yang tinggi.",
+		helix = "Mempercepat tikusmu secara diagonal dengan spiral yang kuat.",
+		dome = "Menciptakan kubah pelindung yang mendorong semua musuh.",
+		lightning = "Memanggil petir kuat yang menyetrum musuh.",
+		superNova = "Membuat supernova yang menghancurkan semua musuh di sekitarmu.",
+		meteorSmash = "Membanting semua musuh seperti ditabrak meteor.",
+		gravitationalAnomaly = "Memulai anomali gravitasi.",
+		deathRay = "Memanggang musuh dengan sinar kematian yang kuat dan misterius.",
+		atomic = "Mengganti ukuran semua pemain dengan acak.",
+		dayOfJudgement = "Menghidupkan kembali semua musuh yang mati, semuanya terhubung satu sama lain.",
+		waterSplash = "Memunculkan tetesan air dari Antartika."
+	},
+	powerType = {
+		atk = "SERANGAN (%d)",
+		def = "PERTAHANAN",
+		divine = "ILAHI"
+	},
+
+	unlockPower = "<FC>[<J>•<FC>] Anda telah mendapatkan kekuatan: %s",
+
+	-- Level names
+	-- @Translator notes: if it has gender variation, { "male", "female" }, else "neutral".
+	levelName = {
+		[000] = "Mutan",
+		[010] = "Necromancer",
+		[020] = "Ilmuwan",
+		[030] = "Titan",
+		[040] = { "Penyihir", "Penyihir Wanita" },
+		[050] = "Pengendali Kenyataan",
+		[060] = { "Penguasa Mantra", "Nyonya Mantra" },
+		[070] = "Pemanggil Perdukunan",
+		[080] = "Penunggang Kuda Wabah",
+		[090] = "Penunggang Kuda Kelaparan",
+		[100] = "Penunggang Kuda Perang",
+		[110] = "Penunggang Kuda Kematian",
+		[120] = "Kekosongan"
+	},
```

Comment:
Can you confirm you have done {'male version', 'female version'} for all titles that could have both genders?

Reply:
For some occupations in Indonesian, it becomes gender-neutral. So I think I've done it all.